modules/kubernetes-addons/aws-efs-csi-driver/locals.tf

```diff
@@ -0,0 +1,88 @@
+locals {
+  name                 = "aws-efs-csi-driver"
+  service_account_name = "efs-csi-sa"
+  namespace            = "kube-system"
+
+  default_helm_config = {
+    name                       = local.name
+    chart                      = local.name
+    repository                 = "https://kubernetes-sigs.github.io/aws-efs-csi-driver/"
+    version                    = "2.2.3"
+    namespace                  = local.namespace
+    timeout                    = "1200"
+    create_namespace           = false
+    values                     = local.default_helm_values
+    set                        = []
+    set_sensitive              = null
+    lint                       = true
+    wait                       = true
+    wait_for_jobs              = false
+    description                = "The AWS EFS CSI driver Helm chart deployment configuration"
+    verify                     = false
+    keyring                    = ""
+    repository_key_file        = ""
+    repository_cert_file       = ""
+    repository_ca_file         = ""
+    repository_username        = ""
+    repository_password        = ""
+    disable_webhooks           = false
+    reuse_values               = false
+    reset_values               = false
+    force_update               = false
+    recreate_pods              = false
+    cleanup_on_fail            = false
+    max_history                = 0
+    atomic                     = false
+    skip_crds                  = false
+    render_subchart_notes      = true
+    disable_openapi_validation = false
+    dependency_update          = false
+    replace                    = false
+    postrender                 = ""
+  }
+
+  helm_config = merge(
+    local.default_helm_config,
+    var.helm_config
+  )
+
+  default_helm_values = [templatefile("${path.module}/values.yaml", {
+    service_account_name = local.service_account_name,
+  })]
+
+  set_values = [
```

Comment:
@vara-bonthu At this stage `values.yaml` is not adding any value any longer, but some of the comments therein. Happy to remove `values.yaml` and move comments here, or leave the file around for future defaults to populate `default_helm_values`.